### PR TITLE
Fix up development logging

### PR DESCRIPTION
**Pull request by agjohnson**
Drop additional unused log files and move logging to the console in development

### CODE REVIEW

File: readthedocs/settings/base.py [hunk 1]
```diff
@@ -333,132 +333,39 @@ def INSTALLED_APPS(self):  # noqa
     SILENCED_SYSTEM_CHECKS = ['fields.W342']
 
     # Logging
-    LOG_FORMAT = "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s"
+    LOG_FORMAT = '%(name)s[%(process)d]: %(levelname)s %(message)s [%(name)s:%(lineno)s]'
```
Comment (humitos) [inline readthedocs/settings/base.py]: `name` is duplicated: take a look at this issue https://github.com/rtfd/readthedocs.org/issues/2851

File: readthedocs/settings/base.py [hunk 2]
```diff
@@ -333,132 +333,39 @@ def INSTALLED_APPS(self):  # noqa
...
     LOGGING = {
         'version': 1,
         'disable_existing_loggers': True,
         'formatters': {
-            'standard': {
+            'default': {
                 'format': LOG_FORMAT,
                 'datefmt': "%d/%b/%Y %H:%M:%S"
```
Comment (humitos) [inline readthedocs/settings/base.py]: maybe we can use a `DATETIME_FORMAT` value for this

File: readthedocs/settings/base.py (hunk 1 above)
Comment (agjohnson) [inline readthedocs/settings/base.py]: Good call, I addressed this in a5389dd

I only added the datetime to the development settings, as we don't need to repeat this in production. I also used the format that would match the output of the http logging.

File: readthedocs/settings/base.py (hunk 2 above)
Comment (agjohnson) [inline readthedocs/settings/base.py]: I'd be interested in this, but the format of DATETIME_FORMAT doesn't seem to be compatible with the logging datetime format.

**Comment by agjohnson** [comment]
I updated this to make things look a bit better -- @humitos look good?

**Comment by humitos** [comment]
So, you want the `asctime` *only* for dev environment? If the answer is yes, it's OK the PR.

**Comment by agjohnson** [comment]
Correct. This is to allow the base `LOG_FORMAT` to be used by our production settings (which doesn't require a timestamp)

**Pull request merged**